src/shapes/Polyline.ts

```diff
@@ -21,12 +21,18 @@ export const polylineDefaultValues: Partial<TClassProperties<Polyline>> = {
   exactBoundingBox: false,
 };
 
-export interface SerializedPolylineProps extends SerializedObjectProps {
+export interface UniquePolylineProps {
   points: XY[];
 }
 
+export interface SerializedPolylineProps
+  extends SerializedObjectProps,
+    UniquePolylineProps {}
+
+export interface PolylineProps extends FabricObjectProps, UniquePolylineProps {}
```

Comment:
this is not correct, i didn't do it on purpose because points is not part of polylineProps. only of the serialized one.

Reply:
Good point. Same for objects in group